contents/python/introduction-to-python.md

```diff
@@ -12,6 +12,21 @@ For absolute beginners, check out [this video](https://www.youtube.com/watch?v=N
 
 Both newbies and experienced programmers can also benefit from [The Python Tutorial](https://docs.python.org/3/tutorial/index.html), which aims to introduce readers to Python's unique features and style.
 
+## Python 2 vs Python 3
+
+Despite obvious similarities in two Python versions, Python 3 intentional backward incompatibility made choosing Python 3 over Python 2 a tough problem for both new and experienced Python programmers. Some concerns include the lack of popular Python 2 packages in Python 3 as well as changes in some Python built-in libraries that might break existing systems. One example is that a simple `print 'Hello World'` will cause runtime error in Python 3 but will run perfectly fine in Python 2.
+
+Read more in this Digital Ocean's [post](https://www.digitalocean.com/community/tutorials/python-2-vs-python-3-practical-considerations-2) to understand this conundrum of choosing between Python 2 and Python 3 better. If you are a new Python programmer, also take a look at these important [changes](https://www.geeksforgeeks.org/important-differences-between-python-2-x-and-python-3-x-with-examples/) to avoid gotchas due to version differences.
```

Comment:
`If you're a new Python programmer` is a weird statement to include in here, as the target audience of these materials are newbies, i.e. those who are new to Python in this case, so it's a little redundant.

You could rephrase the second sentence into `If you choose to learn both Python 2 and Python 3, take a look at some of these important changes to avoid gotchas due to version differences`